framework/doc/content/source/functions/PiecewiseConstant.md

```diff
@@ -8,7 +8,7 @@ The `PiecewiseConstant` function defines the data using a set of x-y data pairs.
 of linearly interpolating between the values, however, the `PiecewiseConstant` function
 is constant when the abscissa is between the values provided by the user.  The `direction`
 parameter controls whether the function takes the value of the abscissa of the
-user-provided point to the right or left of the value at which the function is evaluated.
+user-provided point to the right, left, or centered value at which the function is evaluated.
```

Comment:
needs updating now

Reply:
The `direction` parameter controls whether the function takes the value of the abscissa of the
user-provided point to the `right` or `left` value at which the function is evaluated.
Also available is `right_inclusive` and `left_inclusive`, which will return the value of the function at the specified time.
?

Reply:
yeah. For extra clarity you could add this plot to the docs:
![image](https://user-images.githubusercontent.com/202302/152063265-279ba6c0-63ef-4a69-9641-28d43b9f1e85.png)

Reply:
Yeah doing it right now! Never used the plotly interface, pretty cool